NuGet.Config

```diff
@@ -25,6 +25,9 @@
     <add key="myget.org symreader-portable" value="https://dotnet.myget.org/F/symreader-portable/api/v3/index.json" />
     <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+
+    <!-- Temporary local nuget store to unblock the PR -->
```

Comment:
I'll revert this change once the new NuGet packages are uploaded to NuGet.org (RoslynDependencies.Microsoft.VisualStudio.CodeAnalysis.Sdk.UI and RoslynDependencies.Microsoft.VisualStudio.Diagnostics.PerformanceProvider)